doc/Pacemaker_Explained/en-US/Book_Info.xml

```diff
@@ -6,24 +6,19 @@
   <subtitle>An A-Z guide to Pacemaker's Configuration Options</subtitle>
   <productname>Pacemaker</productname>
   <productnumber>1.1</productnumber>
-  <edition>1</edition>
+  <!--
+	EDITION-PUBSNUMBER should match REVNUMBER in Revision_History.xml.
+	Increment EDITION when the syntax of the documented software
+	changes (pacemaker), and PUBSNUMBER for
+	simple textual changes (corrections, translations, etc.).
+  -->
+  <edition>5</edition>
   <pubsnumber>0</pubsnumber>
   <abstract>
     <para>
       The purpose of this document is to definitively explain the concepts used to configure Pacemaker.
-      To achieve this, it will focus exclusively on the XML syntax used to configure the CIB.
```

Comment:
On 03/02/2015 05:06 PM, Andrew Beekhof wrote:
>>    <pubsnumber>0</pubsnumber>
>>    <abstract>
>>      <para>
>>        The purpose of this document is to definitively explain the concepts used to configure Pacemaker.
>> -      To achieve this, it will focus exclusively on the XML syntax used to configure the CIB.
> 
> Can we have the original paragraph here please.

Sure, but it is duplicated under "The Scope of this Document", which is
the first section of the first chapter:

http://clusterlabs.org/doc/en-US/Pacemaker/1.1-pcs/html/Pacemaker_Explained/ch01.html#_the_scope_of_this_document

I figured it was appropriate to keep it there and shorten the abstract,
but if you want it in both places I can do that.